inspire_schemas/records/seminars.yml

```
@@ -130,6 +133,12 @@ properties:
             Contains a reference to the record replacing the current one, if it
             is marked as :ref:`deleted`.
         title: Record replacing this one
+    material:
```

why not
```suggestion
    material_urls:
```
to be consistent with `join_urls`?